src/util/SimpleTicTacToeGame.ts

```diff
@@ -0,0 +1,69 @@
+import { Move, TileValue } from '../state/AppState';
+import { TicTacToeGame } from './TicTacToeGame';
+import { arePointsEqual } from './Point';
+import { playerToTileValue } from './PlayerToTile';
+
+interface MoveExtended extends Move {
+  value: TileValue;
+}
+
+export class SimpleTicTacToeGame extends TicTacToeGame {
+  constructor(state: TileValue[]) {
+    super([]);
+
+    const simpleMoves = this.getSimpleMoves(state);
+    this.applySimpleMoves(simpleMoves);
+  }
+
+  private getSimpleMoves(state: TileValue[]): MoveExtended[] {
+    if (state.length !== 9) {
+      throw new Error('State has to be of length 9!');
+    }
+
+    let allMoves: MoveExtended[] = [];
+    for (let i = 0; i < state.length; i++) {
+      for (let j = 0; j < 3; j++) {
+        allMoves.push({
+          boardPosition: { x: Math.floor(i / 3), y: i % 3 },
+          tilePosition: { x: Math.floor(j / 3), y: j % 3 },
+          value: state[i],
+        });
+      }
+    }
+
+    return allMoves;
+  }
+
+  private applySimpleMoves(moves: MoveExtended[]) {
+    moves.forEach((move) => {
+      this.applySimpleMove(move);
+    });
+  }
+
+  private applySimpleMove(move: MoveExtended) {
+    const boardToChange = this.board.find((board) =>
```

Comment:
Huh.. I recently rewrote this so that the applyMove code can be shared with the TicTacToeGame class but seems like I never committed this.. :confused: